.github/workflows/tests.yml

```diff
@@ -327,6 +330,36 @@ jobs:
       repository-name: ${{ github.repository }}
       dry-run: false
 
+  build-test-docker-nonroot:
+    uses: ./.github/workflows/build-test-docker.yaml
+    secrets: inherit
+    with:
+      docker-flavor: |
+        latest=auto
```

Comment:
This requires a giant comment saying how important are those lines, and how we can get a big mess if we don't do the right thing here, with some links to some docs.